optuna_dashboard/webpack.config.js

```diff
@@ -41,6 +41,12 @@ var config = {
         test: /\.css$/,
         use: ["style-loader", "css-loader"],
       },
+      {
+        test: /\.m?js$/,
+        resolve: {
+          fullySpecified: false,
+        }
+      }
```

Comment:
I added this change following [this](https://github.com/webpack/webpack/issues/11467#issuecomment-691873586), otherwise we'll get an error: `ERROR in ../tslib/react/pkg/index.js 10:0-208
Module not found: Error: Can't resolve './utils/trialFilter' in 'xxxxxx/optuna-dashboard/tslib/react/pkg'
Did you mean 'trialFilter.js'?
BREAKING CHANGE: The request './utils/trialFilter' failed to resolve only because it was resolved as fully specified`